WooFoundation/WooFoundation/Colors/UIColor+SemanticColors.swift

```diff
@@ -129,10 +126,7 @@ public extension UIColor {
 
     /// Text.
     ///
-    static var textBrand: UIColor {
-        return UIColor(light: .withColorStudio(.wooCommercePurple, shade: .shade60),
-        dark: .withColorStudio(.wooCommercePurple, shade: .shade30))
-    }
+    static var textBrand: UIColor = .accent
```

Comment:
This was matching the previous `accent` color, and I missed updating it in the last PR.
Now I updated to reference `accent` directly instead of duplicating the color definition.